File: 后端/web/src/api/operationRecord.js
import service from '@/utils/request'

// 操作日志接口已切换至去 Sys 化的新路径，后端保留一段旧路径过渡期，若回滚可临时切回
export const deleteOperationRecord = (data) => {
  return service({
    url: '/operation-logs/delete',
    method: 'delete',
    data
  })
}

export const deleteOperationRecordsByIDs = (data) => {
  return service({
    url: '/operation-logs/deleteByIds',
    method: 'delete',
    data
  })
}

export const getOperationRecordList = (params) => {
  return service({
    url: '/operation-logs/list',
    method: 'get',
    params
  })
}

File: 后端/server/router/admin/operation_record.go
package admin

import (
	"github.com/gin-gonic/gin"
)

type OperationRecordRouter struct{}

func (s *OperationRecordRouter) InitOperationRecordRouter(Router *gin.RouterGroup) {
	operationRecordRouter := Router.Group("operation-logs")
	{
		operationRecordRouter.DELETE("delete", operationRecordApi.DeleteOperationRecord)            // 删除操作日志
		operationRecordRouter.DELETE("deleteByIds", operationRecordApi.DeleteOperationRecordsByIDs) // 批量删除操作日志
		operationRecordRouter.GET("detail", operationRecordApi.FindOperationRecord)                 // 根据ID查询操作日志
		operationRecordRouter.GET("list", operationRecordApi.GetOperationRecordList)                // 分页获取操作日志列表
	}
}
